programs/drift/src/controller/orders.rs

```diff
@@ -1704,8 +1714,17 @@ fn fulfill_perp_order(
                 MarginRequirementType::Maintenance
             } else {
                 MarginRequirementType::Fill
-            }),
-        )?;
+            })
+            .fuel_perp_delta(market_index, taker_base_asset_amount_signed as i64),
```

Comment:
when do you need to add fuel_numerator vs not add it?

Reply:
should always add the fuel numerator actually (to track time elapsed)

doing a scan across for that